docs/providers/aws/events/http-api.md

```diff
@@ -392,6 +392,18 @@ provider:
     payload: '1.0'
 ```
 
+Also, the payload version could be configured in function level as follows:
+
+```yaml
+functions:
+  hello:
+    handler: index.handler
+    httpApi:
+      payload: '1.0'
+    method: GET
```

Comment:
I believe the listing here is not correct it doesn't configure a valid event. 

It should rather look similar to:
```
functions:
  hello:
    handler: index.handler
    httpApi:
      payload: '1.0'
    events:
      - httpApi:
          path: /hello
          method: GET
```